src/main/java/uk/gov/pay/api/model/directdebit/agreement/CreateMandateRequest.java

```diff
@@ -5,19 +5,32 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
+import org.hibernate.validator.constraints.Length;
```

Comment:
Just a note, it might be better to use `@Size` which is included in the validation spec `javax.validation.constraints` and is implementation agnostic. Hibernate `@Length` is hibernate specific version of `@Size` and it ties us to use hibernate. Bit more info here: https://www.baeldung.com/jpa-size-length-column-differences

Reply:
fixed

Reply:
can you use Size for the other fields too?

Reply:
argh i got confused. i've used Size all over now.